draft-js-counter-plugin/src/__test__/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
I would recommend to split up this test file & have a separate test in every component folder. The idea is to find the related test in the folder where the actual code is.